### PR TITLE
MINIFICPP-1405 - Update civetweb version, remove patch, disable building lua and duktape deps

### PR DESCRIPTION
**Pull request by hunyadi-dev**
Cherry picking the civetweb fix onto the release candidate branch.

### CODE REVIEW

**Comment by fgerlits** [comment]
Same as #952, which is already in main, so should be safe to merge.

**Comment by fgerlits** [comment]
You will also need to cherry-pick #925 because of the `set-env` deprecation in GitHub Actions.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -259,8 +249,7 @@ jobs:
             ubuntu-16.04-shared-ccache-refs/heads/main-
       - id: install_deps
         run: |
-          echo "::set-env name=CMAKE_BUILD_OPTIONS::-DENABLE_PCAP=TRUE"
           sudo apt install -y ccache
-          echo "::set-env name=PATH::/usr/lib/ccache:$PATH"
+          echo "PATH=/usr/lib/ccache:$PATH" >> $GITHUB_ENV
       - id: build
         run: ./bootstrap.sh -e -t && cd build  && cmake -DUSE_SHARED_LIBS=ON -DSTRICT_GSL_CHECKS=AUDIT .. &&  cmake --build . --parallel 4  && make test ARGS="--timeout 300 -j4 --output-on-failure"
```
Comment (fgerlits) [inline .github/workflows/ci.yml:255]: It looks like you manually removed the docker timeout change from this cherry-picked commit.  Why?

It's OK if you have a good reason, but there is a danger that this change will be accidentally reverted when this branch is merged back to main.

Also, if you remove this part of the commit, please remove it from the commit message, too -- or, even better, mention in the commit message that you have removed this.

**Review by fgerlits — APPROVED** [review]

**Review by arpadboda — APPROVED** [review]
LGTM, thanks!
The OSX failures are not related, merging.

**Comment by arpadboda** [comment]
Merged in scope of https://github.com/apache/nifi-minifi-cpp/commit/24d16c1c92d1bf818e5cbba7e91f7eb0688dfcb2

**Pull request closed without merging**